lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
> Would it not be simpler to make Travis check that the manual can be built?

There are several aspects here. Perhaps this should be an enhancement proposal, 
not a hidden side note in the commit, but for now I'll put it here:

* what do we want to test?
a) that 'make manuals' does not fail (a common case when it fails
is an error in GAPDoc syntax)
b) that there are no unresolved references in the manual
c) that manual examples run as expected (with default selection of packages).

We run nightly tests using Jenkins CI where failures of types
(a) and (c) are detected automatically and trigger build failure.
We may discuss which of these checks we want to have in Travis.
Perhaps at least (a) would be important to have, to check pull
requests related to GAP manuals.